frontend/src/pages/CollaborationPage.js

```diff
@@ -12,14 +12,18 @@ import ReviewPartnerDialog from '../components/ReviewPartnerDialog'
 import { getCollabRoomId } from '../utils/main'
 import { expirationCheck } from '../utils/main'
 import { collabSocket, matchingSocket } from '../utils/socket'
+import moment from 'moment'
 
 const CollaborationPage = () => {
   const location = useLocation()
   const navigate = useNavigate()
 
+  const TENSECONDS = 10 * 1000
```

Comment:
This is to convert 10 secs to milliseconds? Perhaps a better naming convention is `TEN_SECONDS_IN_MILLISECONDS`?